config/cesm/machines/config_machines.xml

```diff
@@ -433,6 +433,63 @@ This allows using a different mpirun command to launch unit tests
     </resource_limits>
   </machine>
 
+  <machine MACH="coeus">
+    <DESC>
+      Portland State University Coeus Cluster CentOS 7
+    </DESC>
+    <NODENAME_REGEX>(login[1,2].cluster|compute[0-9]*.cluster)</NODENAME_REGEX>
+    <OS>LINUX</OS>
+    <COMPILERS>gnu</COMPILERS>
+    <MPILIBS>mvapich2</MPILIBS>
+    <PROJECT>none</PROJECT>
+    <CIME_OUTPUT_ROOT>$ENV{CESMDATAROOT}/$USER</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>$ENV{CESMDATAROOT}/inputdata</DIN_LOC_ROOT>
+    <DOUT_S_ROOT>$ENV{CESMDATAROOT}/archive/$CASE</DOUT_S_ROOT>
+    <BASELINE_ROOT>$ENV{CESMDATAROOT}/cesm_baselines</BASELINE_ROOT>
+    <CCSM_CPRNC>/vol/apps/hpc/src/cesm-2.1.0/cime/tools/cprnc/cprnc</CCSM_CPRNC>
+    <GMAKE>make</GMAKE>
+    <GMAKE_J>8</GMAKE_J>
+    <BATCH_SYSTEM>slurm</BATCH_SYSTEM>
+    <SUPPORTED_BY>oit-rc-groups@pdx.edu</SUPPORTED_BY>
+    <MAX_TASKS_PER_NODE>19</MAX_TASKS_PER_NODE>
+    <MAX_MPITASKS_PER_NODE>19</MAX_MPITASKS_PER_NODE>
```

Comment:
19 is a very strange number for tasks per node - what type of machine is this?

Reply:
Thank you for all your feedback. Our compute nodes are Dual Intel Xeon E2630 v4, 10 cores each. I have heard mixed advice on if this should be kept 1 below the number of cores, should I bump this up to 20?

Reply:
There is occasionally good reason to leave a core available for the OS but we would typically set
MAX_MPI_TASKS_PER_NODE=20 and MAX_TASKS_PER_NODE=40 - (this allows for smt threads which may improve performance in some cases) Try it with a case or two and see what works best for you.  The PFS test is designed to measure performance of a given compset.